uk.co.compucorp.civicrm.hrleaveandabsences/scss/components/_manager_calendar.scss

```diff
@@ -30,10 +30,6 @@
 
     .ui-select-container {
```

Comment:
do we still need this?

Reply:
Yes we need it, it has the `height` property.